### ISSUE TITLE
Bluetooth support

### ISSUE THREAD

**Issue by langovoi** [espressif/esp-apple-homekit-adk#1]
Do you have plan to make bluetooth support?

**Comment by shahpiyushv** [comment]
@langovoi , we do have plans to add it, but not any time soon. We are open to contributions though :)

**Comment by HomeKidd** [comment]
> @langovoi , we do have plans to add it, but not any time soon. We are open to contributions though :)

BLE support with deepsleep would be great! 😄

**Comment by dakhnod** [comment]
So, did anything happen?

I can see a "#if (BLE)", but will defining BLE suffice in order to enable BLE support?

**Comment by HomeKidd** [comment]
@dakhnod i dont think so, last commit was at May 14. That definition for BLE is just part of HomeKit-ADK😉

**Comment by shahpiyushv** [comment]
@dakhnod , what @HomeKidd said is (unfortunately) correct.

**Comment by crasmussen** [comment]
Hi @shahpiyushv , any estimate on when BLE might be supported?

**Comment by mriksman** [comment]
Super bummed to find that setting `define BLE 1` did not compile.
```
../main/app_main.c:130:17: error: 'HAPPlatformBLEPeripheralManagerOptions' {aka 'struct <anonymous>'} has no member named 'keyValueStore'
     blePMOptions.keyValueStore = &platform.keyValueStore;
```

+1 for BLE support.

**Comment by yrjyrj123** [comment]
Is there any update?
